examples/sw_williamson2_triangle.py

```diff
@@ -16,7 +16,7 @@
 mesh.init_cell_orientations(global_normal)
 
 timestepping = TimesteppingParameters()
-output = OutputParameters()
+output = OutputParameters(dumplist=(True,True), dirname='sw_williamson2')
```

Comment:
Yes.


________________________________
From: David A. Ham <notifications@github.com>
Sent: 21 March 2016 11:25
To: firedrakeproject/dcore
Cc: Shipton, Jemma
Subject: Re: [dcore] make a new directory for dumping results, checking first that is does… (#24)


In examples/sw_williamson2_triangle.py<https://github.com/firedrakeproject/dcore/pull/24#discussion_r56808201>:

> @@ -16,7 +16,7 @@
>  mesh.init_cell_orientations(global_normal)
>
>  timestepping = TimesteppingParameters()
> -output = OutputParameters()
> +output = OutputParameters(dumplist=(True,True), dirname='sw_williamson2')


Wouldn't it be much more intelligible to name the fields you are dumping?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/firedrakeproject/dcore/pull/24/files/09228c50c4993b66d733ff3866009ab879fcc255#r56808201>